src/python/bot/fuzzers/syzkaller/fuzzer.py

```diff
@@ -14,70 +14,61 @@
 """syzkaller fuzzer."""
 from __future__ import absolute_import
 from base import utils
-from bot.fuzzers import builtin
+from bot.fuzzers import engine
 from bot.fuzzers import utils as fuzzer_utils
-from bot.fuzzers.syzkaller import constants
-from bot.fuzzers.syzkaller import generate_config
-from builtins import object
+from bot.fuzzers.syzkaller import config
 from system import environment
 from system import new_process
 import copy
+import os
 import re
 import tempfile
 
 # Regex to find testcase path from a crash.
 KASAN_CRASH_TESTCASE_REGEX = (r'.*Test unit written to\s*'
                               r'(Read|Write) of .*')
+SYZKALLER_WORK_FOLDER = '/tmp/syzkaller'
+VMLINUX_FOLDER = '/tmp/syzkaller/vmlinux'
 
 
-def get_arguments(fuzzer_path):
+def get_arguments(unused_fuzzer_path):
   """Get arguments for a given fuzz target."""
-  del fuzzer_path
   build_dir = environment.get_value('BUILD_DIR')
   device_serial = environment.get_value('ANDROID_SERIAL')
-  json_config_path = '/tmp/' + device_serial + '/config.json'
-  generate_config.run(
+  json_config_path = os.path.join('/tmp', device_serial, 'config.json')
+  config.generate(
       serial=device_serial,
-      work_dir_path='/tmp/syzkaller',
-      binary_path=build_dir + '/syzkaller',
-      vmlinux_path='/tmp/syzkaller/vmlinux',
+      work_dir_path=SYZKALLER_WORK_FOLDER,
+      binary_path=os.path.join(build_dir, 'syzkaller'),
+      vmlinux_path=VMLINUX_FOLDER,
```

Comment:
Who downloads this?

Reply:
This is downloaded by the tradefed client ahead of fuzzing.